src/main/info-initializer.cpp

```diff
@@ -111,12 +112,12 @@ static errr init_info(std::string_view filename, angband_header &head, InfoType
     char buf[1024]{};
     const auto err = init_info_txt(fp, buf, &head, parser);
```

Comment:
この後 `std::get<0>(err)`、`std::get<1>(err)` が複数回でてくるので、以下のように`error_code`と`errro_line`に構造化束縛により最初に展開しておき、それを使用したほうが可読性が上がると思います。

```suggestion
    const auto &[error_code, error_line] = init_info_txt(fp, buf, &head, parser);
```